docs/concepts/moneyd.md

```diff
@@ -0,0 +1,19 @@
+# moneyd
+[`moneyd`](https://github.com/interledgerjs/moneyd) is a daemon that enables you to connect to an Interledger node, 
+and to send or receive value in the form of ILP packets to or from that node. `moneyd` functions like your 
+home wifi router in the sense that both transfer packets and connect to an upstream network. However, `moneyd` uses 
```

Comment:
in the sense that both -> in the sense that it both

Reply:
Here 'both' is referring to moneyd and home router.